keep-only-warm-data/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+
+services:
+  kuzzle:
+    image: kuzzleio/howto-keepwarmdata-kuzzle
+    command: /custom_start.sh
+    volumes:
+      - "./scripts:/scripts"
+    depends_on:
+      - redis
+      - elasticsearch
+    ports:
+      - "8080:8080"
+      - "9229:9229"
+      - "7512:7512"
+    environment:
+      - kuzzle_services__db__client__host=http://elasticsearch:9200
+      - kuzzle_services__internalCache__node__host=redis
+      - kuzzle_services__memoryStorage__node__host=redis
+      - kuzzle_server__maxRequestSize=1GB
+      - kuzzle_limits__documentsWriteCount=100000
+      - NODE_ENV=development
+
+  redis:
+    image: redis:3.2
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.4.1
+    ports:
+      - "9200:9200"
+    environment:
```

Comment:
- I'd rather use kuzzleio elasticsearch image, which fixes the jvm options not being taken into account
- Do we need to expose elasticsearch to the host?

Reply:
Sometimes I test my scripts directly from my host, so is much convenient for me to expose elastic to the host too.